src/Components/Game/Game.js

```diff
@@ -81,9 +81,8 @@ export default function Game() {
           <ScamModal
             closeModal={setModalClose}
             result={result}
-            score={score}
             questionCategory={questionCategory}
-            hints={hints}
+            hint={roundQuestions[questionIndex - 1].hint}
```

Comment:
We advance the question counter before the user clicks "next" which is slightly awkward, but this functions as a workaround.